woocommerce/woocommerce-ios
# Dynamic Dashboard: Display top performer list in top performers dashboard card

Part of #12487 


## Description

This PR updates the top performers dashboard card with the existing top performers list. Changes include:
- Renamed `DashboardTopPerformersView` to `TopPerformersPeriodView` to avoid ambiguity.
- Added the top performers list to the dashboard card view.
- Updated the top performers dashboard view model to sync data and load from storage.
- Updated the dashboard view model to reload stats data when the cards are enabled.

## Testing instructions

- Enable the `dynamicDashboard` feature flag and build the app.
- Log in to a store and enable the Top Performers card if needed.
- Switch between different time ranges, confirm that list of top earner items are loaded and displayed correctly.
- Tap on any item, confirm that the relevant product detail screen is displayed.

## Screenshots

<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/c21db4a2-5e60-4584-86dc-50c053d3c51f" width=320 />


---
